src/api/poi/index.ts

```diff
@@ -25,8 +25,7 @@ import {
 
 import apiSlice from "..";
 import { generateImageStorageRef } from "../../utils/firebase/storage";
-import { getDownloadURL, getStorage, ref, uploadBytes } from "firebase/storage";
-import { firebaseApp } from "../../utils/firebase";
+import { uploadBytes } from "firebase/storage";
```

Comment:
reorder

Reply:
`import { getDownloadURL, getStorage, ref, uploadBytes } from "firebase/storage";` alreay deleted no need to reorder